pkg/node/devnode.go

```diff
@@ -278,7 +278,10 @@ func NewDevBee(logger log.Logger, o *DevOptions) (b *DevBee, err error) {
 				}
 				topUpAmount := big.NewInt(0).Div(batch.Value, big.NewInt(int64(1<<(batch.Depth))))
 
-				post.HandleTopUp(batch.ID, topUpAmount)
+				err = post.HandleTopUp(batch.ID, topUpAmount)
```

Comment:
This is wrong. This function doesnt have to change.

All you need to do is return error here. Not do any calculations. Same for HandleDepthIncrease.